authlib/auth.py

```diff
@@ -14,10 +15,15 @@
 ENC_NONCE = osenv.get('ENC_NONCE')
 
 STORAGE = osenv.get('STORAGE', 'SQLITE')
+COOKIE_NAME = osenv.get('COOKIE_NAME')
 store = None
 
 # ------------------------------------------------------------------------------
 
+@st.cache(allow_output_mutation=True)
```

Comment:
I'd use `@st.experimental_memo` here.

Reply:
Thanks, but since experimental_memo did not work, I went with experimental_singleton, which seemed more appropriate. This one works fine and seems appropriate.